metalearn/metafeatures/metafeatures.py

```diff
@@ -389,7 +396,11 @@ def _get_preprocessed_data(self, X_sample, X_sampled_columns, column_types, seed
             )
             if column_types[feature_series.name] == self.CATEGORICAL:
                 feature_series = pd.get_dummies(feature_series)
-            series_array.append(feature_series)
+            elif column_types[feature_series.name] == self.TEXT:
+                X_sample.drop(columns=[feature])
```

Comment:
What does dropping the column do? It doesn't look like it affects anything and doesn't it change the columns that are being looped over?

Reply:
@poolguy fixed this in commit bfbb5cc85b896bbe5ef031921356cbae921f6f06